### DISCUSSION TITLE
Why "sys.prefix", "sys.exec_prefix" is None?

### DISCUSSION THREAD

**Discussion by kazu8-narita** [pythonnet/pythonnet#1757] [Q&A]
### Environment

-   Pythonnet version: 2.5.2 ("pythonnet_netstandard_py38_win" from nuget)
-   Python version: Python 3.8.1
-   Operating System: Windows10 Pro(21H2)
-   .NET Runtime: .net5.0

### Details

- I tried print site.getsitepackages() from my python script.

```python
# myscript.py
import site
import pprint
print("site.getsitepackages", site.getsitepackages()) # expected got paths! ex) ['C:/~~/site-packages', 'c:/~~/venv']
```

- Succeed to I got paths when executed from python interpreter directly.

```
cmd > python.exe .\myscript.py
site.getsitepackages ['C:/~~/site-packages', 'c:/~~/venv']
```

- However, I got nothing when used Pythonnet from csharp app...

```
cmd > myapp.exe
site.getsitepackages []
```

  * csharp app is this.(console app)

```csharp
  // myapp.exe
  static void Main(string[] args)
  {
      string[] pathes = {
          PythonEngine.PythonPath,
          Path.GetFullPath("./pysrc"), //script location
      };
      PythonEngine.PythonPath = string.Join(Path.PathSeparator.ToString(), pathes);
      PythonEngine.PythonHome = @"C:/Users/xxx/AppData/Local/Programs/Python/Python38";

      PythonEngine.Initialize();
      using (Py.GIL())
      {
          PythonEngine.Exec("import myscript");
      }
    PythonEngine.Shutdown();
  }
```

- I found python's site package depends `sys.prefix`, `sys.exec_prefix`.  
  So I printed it ... but prints none. (from csharp only failed. when executes python interpreter directly, printed paths)

```python
# myscript.py
import sys
print('sys.executable', sys.executable)
print('sys.prefix', sys.prefix)
print('sys.exec_prefix', sys.exec_prefix)
print('sys.base_prefix', sys.base_prefix)

# all returns None, when used Pythonnet!
```

Why is this? 
what am I missing something??

**Comment by filmor** [comment]
`pythonnet_netstandard_py38_win` is an unsupported fork. Do you get the same issue on our version?

**Reply by kazu8-narita** [reply]
@filmor thanks comment:)

> Do you get the same issue on our version?

Yes, I had to same issue at supported package...
as follows..

* install supported package from nuget 
  * [pythonnet - 3.0.0-preview2022-04-11](https://www.nuget.org/packages/pythonnet/)
  * (removed `pythonnet_netstandard_py38_win` before install)

* retried to run my csharp app
```cs
  // myapp.exe
  static void Main(string[] args)
  {
      Environment.SetEnvironmentVariable("PYTHONNET_PYDLL", @"C:/Users/xxx/AppData/Local/Programs/Python/Python38/python38.dll", EnvironmentVariableTarget.Process);
      string[] pathes = {
          PythonEngine.PythonPath,
          Path.GetFullPath("./pysrc"), //script location
      };
      PythonEngine.PythonPath = string.Join(Path.PathSeparator.ToString(), pathes);
      //PythonEngine.PythonHome = @"C:/Users/xxx/AppData/Local/Programs/Python/Python38";

      PythonEngine.Initialize();
      using (Py.GIL())
      {
          PythonEngine.Exec("import myscript");
      }
    PythonEngine.Shutdown();
  }
```

* site.getsitepackage() is empty array and sys.*** variables is None.
```
sys.prefix
sys.exec_prefix
sys.base_prefix
sys.base_exec_prefix
site.getsitepackages []
```
